deploy/local/vars_lnx.sh

```diff
@@ -223,6 +224,17 @@ if [ ! -d $__daal_tmp_dir ]; then
     __daal_tmp_dir=${component_root}
 fi
 
+ARCH=$(uname -m)
```

Comment:
Since this function is sourced, any use of ARCH/ARCH_DIR in environments that call this script would be overwritten. Would prefer changing ARCH and ARCH_DIR to ARCH_ONEDAL and ARCH_DIR_ONEDAL to avoid, as ARCH is a common env variable.